src/main/webapp/js/components/interface/drawer/TabbedDrawer.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Tabbed Drawer resizable component
+ * It uses the components DrawerButton and Rnd to create a resizable Tabbed Drawer.
+ * 
+ *  @author Dario Del Piano
+ */
+
+define(function (require) {
+
+    var React = require('react');
+    var DrawerButton = require('./DrawerButton');
+    var Rnd = require('react-rnd').default;
+    require('./TabbedDrawer.less');
+
+    class TabbedDrawer extends React.Component {
+        constructor(props) {
+            super(props);
+            
+            // the state drawerOpened is used to keep track of the display css attribute
+            // the buttonSelected tell us what child has to be displayed
+            // the drawerHeight is used for the resizing
+            this.state = {
+                drawerOpened: false,
+                buttonSelected: null,
+                drawerHeight: 250
+            }
+            this.openDrawer = this.openDrawer.bind(this);
+            this.renderMyLabels = this.renderMyLabels.bind(this);
+            this.renderMyChilds = this.renderMyChilds.bind(this);
+            this.drawerResizing = this.drawerResizing.bind(this);
+            this.drawerStopResizing = this.drawerStopResizing.bind(this);
+            this.closeDrawer = this.closeDrawer.bind(this);
+            this.maximizeDrawer = this.maximizeDrawer.bind(this);
+            this.minimizeDrawer = this.minimizeDrawer.bind(this);
+        }
```

Comment:
What's the difference between close and minimize?

Reply:
just updated, minimize keeps the same height we had, close restore this to 250px. thx :)